modules/layers/src/scatterplot-layer/scatterplot-layer-uniforms.ts

```diff
@@ -0,0 +1,56 @@
+import {ShaderModule, UniformFormat} from '@luma.gl/shadertools';
+const uniformBlock = `\
+uniform scatterplotUniforms {
+  uniform float radiusScale;
+  uniform float radiusMinPixels;
+  uniform float radiusMaxPixels;
+  uniform float lineWidthScale;
+  uniform float lineWidthMinPixels;
+  uniform float lineWidthMaxPixels;
+  uniform float stroked;
+  uniform bool filled;
+  uniform bool antialiasing;
+  uniform bool billboard;
+  uniform highp int radiusUnits;
+  uniform highp int lineWidthUnits;
+} scatterplot;
+`;
+
+type DefaultUniformTypes<PropsT extends {} = {}> = {
```

Comment:
@Pessimistress is this the sort of type cross you had in mind?

Reply:
[Playground](https://www.typescriptlang.org/play/?#code/C4TwDgpgBAqgdgSwGYHsBOBbACmlYDOUAvFAN4BQUVUA2nAIYYQBcU+waCcA5gLqtwArhgBGENFAA+UEShQAbCPThSoQ0eJq9yAXwDc5cqEhQAcsICCaNPRAAmKiTrCxaADRqX43gePRzGFY2IADMjrTqrh6R4tFeaD5G4P6W1rYALOHOGu6eOXH5ea6JfmapwQBsWTG5NQVRRbGNtfElyWWBaSAAHNXx9U11zQMthUM1bSYBQbYAnH1j-cPL40urizmTKZ3BAIwOTusNR4NrZxvH55cX3r7t0127VYdXpzej159vXx-ff78AkYjRJJEzwZDoDAAFWSAB4AGr0eSCCBQqAQAAewAgcAAJoQaqpZAolCppDUtAA+YiUKCI5Go9FYnH45pEuSKZRQAD8UAAFAByJAhOwC1QChAisXSAWCKUASlprHpKLRmOxeMID2CDl5goAbhAAMZ2WHCuyU6VQAWGk2wyUWq0242muWOxVUZVI1VMjWs7W2MJ651GkJmkWW8W2sMOyMy6Owt2Wj1QL0MtXMzUdGYgTLB23pcOOqPGwuxp0FxMRp0YejAOwY03m5NKunexnqlla8q2KrB2v1jFh5s1ushRtFluetvp31d7NdXr9usNwsj8UD9ITkcptM+ztZgMgea8gUD8fD6utlUdzP+nsgfY8-lnscYtfVjd1reX93X9sZn63Y7LYTzPq+wBbh+jqtnAECGmgBigtA4KoJgMKQPgsI4Hg+CAfOqGQjhBDUiQFDUBEjDQFwUAANYQCAKBIFAxF4fwsCIGh0JwqxUJ0FRvCUgYOiGKUAAytjiIRmCscQZC0vgCAAF4QAAykaSIQNyAjxAYVCKSpACyXBYAgGIQPI+Dac0elsMpECGfQGKmeZlnWTUtkGRA4LAFZOk5LZRoKOg1nZD8QLNIkVBIAg8iKLi1nEpycDCYYQVwOwUCCJxkIYRA+CsNJ3GYbCABKEAAI6CAgaAQLisISSAUk5TJuAkaR8n6fZ6maawQpSm4Cn2cZcAuRZBXWuaAqDV1RlOWNll9VNM12SpPkTRKA20jFcV1UtW1UEF8joH1lYjroBhAA)

Reply:
Nice, integrated into PR. Thanks 😄